podman/api/ssh.py

```diff
@@ -202,6 +202,8 @@ def connect(self) -> None:
 class SSHConnectionPool(urllib3.HTTPConnectionPool):
     """Specialized HTTPConnectionPool for holding SSH connections."""
 
+    # pylint complains if attributes aren't all caps.  Nobody will ever fix this.
+    # pylint: disable=invalid-name
     ConnectionCls = SSHConnection
```

Comment:
```suggestion
    ConnectionCls = SSHConnection        # pylint: disable=invalid-name
```
Would be less invasive. Otherwise, I believe you are turning off checking for the remainder of the file.

Reply:
Thanks, fixed and will rebase and force-push.